connection.js

```diff
@@ -731,7 +780,7 @@ class Connection {
                             universalPeriodtimeToString = timespanToDisplayString(universalPeriodtimeNumber);
                         }
                         // then we add the wanted explanation for canadian number of stones
-                        canadianPeriodtimeSentence = `for all the ${t.stones_per_period} stones`; // e.g. "12 seconds per stone, same as 300 seconds for all the 25 stones"
+                        canadianPeriodtimeSentence = "for all the " + t.stones_per_period + " stones"; // e.g. "12 seconds per stone, same as 300 seconds for all the 25 stones"
```

Comment:
This was in string interpolation style already, why remove it? If anything we should add string interpolation elsewhere as it's more readable.

Reply:
this has been removed in UHMAEAT 2.2 actually

all the more reason to merge this and go all the way down until end of #129 , then we can review on top of it